### DISCUSSION TITLE
Why Parts' active is still true?

### DISCUSSION THREAD

**Discussion by Alex-Cheng** [ClickHouse/ClickHouse#37238] [Q&A] [answered]
After run the command `optimize table events`, I query the parts of table events with the SQL `select partition,name, active, rows, level,partition_id, path from system.parts where active=1 and table='events'` and got the results as following.

![image](https://user-images.githubusercontent.com/1518453/168542927-e6d1ef0e-ed21-46b0-9149-e0086cb791a5.png)

It is strange that the parts `all_1_9_4`, `all_1_6_1`, `all_7_7_0`, `all_8_8_0`, `all_9_9_0` are all active, and the sum of rows number of `all_1_6_1`, `all_7_7_0`, `all_8_8_0` and `all_9_9_0` are equal to the total rows number of table `events` i.e. 10M which is the rows number of part  `all_1_9_4`. 

By debugging I found that actually the part to be read is only the part `all_1_9_4`, but why the rest are still active=1? Is it a bug?

Thank you in advance.

**Comment by SaltTan** [comment, marked as answer]
You should add `database` to your query.

**Reply by Alex-Cheng** [reply]
Yes, I noticed that there are two tables with same names in different databases.
